include/Fader.h

```diff
@@ -145,10 +169,16 @@ class LMMS_EXPORT Fader : public QWidget, public FloatModelView
 
 	QPixmap m_knob {embed::getIconPixmap("fader_knob")};
 
+	// Stores the offset to the knob center when the user drags the fader knob.
+	// This is needed to make it feel like the users drag the knob without it
+	// jumping immediately to the click position.
```

Comment:
```suggestion
	//! Stores the offset to the knob center when the user drags the fader knob.
	//! This is needed to make it feel like the users drag the knob without it
	//! jumping immediately to the click position.
```

Reply:
Switched to a Doxygen style block comment with commit 8d5c5230a46.